src/adapter/src/coord.rs

```diff
@@ -474,13 +474,9 @@ impl<S: Append + 'static> Coordinator<S> {
         // This is disabled for the moment because it has unusual upper
         // advancement behavior.
         // See: https://materializeinc.slack.com/archives/C01CFKM1QRF/p1660726837927649
-        let status_collection_id = if false {
-            Some(self.catalog.resolve_builtin_storage_collection(
-                &crate::catalog::builtin::MZ_SOURCE_STATUS_HISTORY,
-            ))
-        } else {
-            None
-        };
+        let source_status_history_shard = self
```

Comment:
We tend to not use the term "shard" outside of storage proper, because the "outside" world is not meant to know about shards, so `source_status_history_collection_id` (a mouthful!) or sth like it seems better

Reply:
IIUC "collection id" often refers to the global id, which this status history collection also has of course. I'm a bit worried about introducing ambiguity!

Reply:
It turns out I am the one who has confused myself! Making this change.